composer.json

```diff
@@ -111,6 +111,30 @@
                     "type": "zip"
                 }
             }
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "library-ckeditor/colordialog",
+                "version": "4.7.0",
+                "type": "drupal-library",
+                "dist": {
+                    "url": "http://download.ckeditor.com/colordialog/releases/colordialog_4.7.0.zip",
+                    "type": "zip"
+                }
+            }
+        },
+        {
+            "type": "package",
```

Comment:
These repositories should be added to https://github.com/ymcatwincities/openy-project as well in separate PR.